example-i3blocks.conf

```diff
@@ -15,6 +15,8 @@ markup=none
 [public-ip]
 label=
 command=~/.config/i3blocks-go/public-ip
+# To specify interface to monitor, pass interface name
```

Comment:
Please add your hint for specifying the ```-ifaceName``` argument to the main comment for this module in lines https://github.com/numbleroot/i3blocks-go/blob/1ac990dd9b86653ece28b32b78619496fe86e8ed/example-i3blocks.conf#L13
and remove these two lines. Also, please adjust the comment to not reference the old URL anymore.